test/compiler/inline.jl

```diff
@@ -275,3 +283,15 @@ let ci = code_typed(f34900, Tuple{Int, Int})[1].first
     @test length(ci.code) == 1 && isexpr(ci.code[1], :return) &&
         ci.code[1].args[1].id == 2
 end
+
+@testset "check jl_ast_flag_inlineable for inline macro" begin
+    ret = ccall(:jl_ast_flag_inlineable, Bool, (Any,), first(methods(@inline x -> x)).source)
+    @test ret == true
```

Comment:
The `ret` variable is redundant here; can just do `@test ccall(...` etc. It is also more idiomatic to use `@test x` and `@test !x` instead of `@test x == true`.

Reply:
fixed in 57ecb07